dev/Dockerfile

```diff
@@ -2,14 +2,19 @@
 FROM minutes-prod:latest
 
 # Audio codec.
-RUN apt-get install -y libav-tools curl
+RUN apt-get install -y libav-tools curl python-dev
 
 # Add additional development requirements.
 ADD ./dev/requirements.txt /env/requirements-dev.txt
 RUN pip install -r /env/requirements-dev.txt
 
+# Additional things we install with conda.
+RUN conda install psutil nodejs
```

Comment:
For some reason, my build fails with `/bin/sh: 1: conda: not found` here, even though our `minutes-prod` image should be based on `continuumio/miniconda3`. Strange…

Reply:
So the issue ended up being that my local `minutes-prod` image was too old. As long as you pull the latest version of our base image, this seems to build successfully.